app/models/pillar.rb

```diff
@@ -2,12 +2,18 @@
 # rubocop:disable Metrics/ClassLength
 class Pillar < ApplicationRecord
   validates :pillar, presence: true
-  validates :value, presence: true
+  validates :value, presence: true, eviction: true
```

Comment:
This is why I never liked this flat model that we needed when the `pillars` table was shared with salt (nowadays this is no longer the case). I'd say moving this to an explicit model would be cleaner so we can properly add the validation for that model specifically, but to be honest... at this point this is still fine, and probably not worth the refactor.